packages/api/src/authentication/authentication.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { BadRequestException, Body, Controller, Post, Req, UseGuards } from "@nestjs/common";
+import { User } from "@cryptify/common/src/entities/user";
+import { signUpSchema } from "@cryptify/common/src/validations/user";
+import { AuthenticationService } from "@cryptify/api/src/authentication/authentication.service";
+import { AuthGuard } from "@nestjs/passport";
+
+@Controller("auth")
+export class AuthenticationController {
+    constructor(private readonly authService: AuthenticationService) {}
+
+    @Post("signup")
+    async signUp(@Body() user: User): Promise<any> {
+        const { error } = signUpSchema.validate(user);
```

Comment:
I think this is best practice no?

```suggestion
        const { value, error } = signUpSchema.validate(user);
```

Reply:
right was using the user var to create the new user through the authService but it makes more sense to use "value"